### ISSUE TITLE
error Pusher.ScriptReceivers[1] is not a function

### ISSUE THREAD

**Issue by wahyupermadie** [jaouadballat/chatapp#4]
hello i want to ask why i got error Pusher.ScriptReceivers[1] is not a function while i type a chat.
i follow all of your tutorial.
but i got this error and trying to fix but i cant.

thanks

**Comment by jaouadballat** [comment]
Are you using two or more instances of the library in your project? . The reason is that the library makes use of the global namespace (window.Pusher) for some of its functionality and if a second instance is loaded it will override any previous instances.

**Comment by wahyupermadie** [comment]
can u help me how to do that ?

**Comment by jaouadballat** [comment]
wish version of laravel you use ?

**Comment by wahyupermadie** [comment]
5.6 sir this my code https://github.com/wahyupermadie/laraChat-Vue-Pusher

**Comment by jaouadballat** [comment]
in bootstrap.js you have 
`import Pusher from 'pusher-js'
import Echo from 'laravel-echo'
window.Pusher = require('pusher-js');`
as you can see you have imported pusher twice delete the first line and try again

**Comment by wahyupermadie** [comment]
import Echo from 'laravel-echo'

window.Pusher = require('pusher-js');

still got the same error sir

**Comment by jaouadballat** [comment]
have you installed dependencies ??

**Comment by wahyupermadie** [comment]
yes of course i follow all of ur step, can u help me by cloning my code and
check at your pc ?

**Comment by jaouadballat** [comment]
now i'm using just phone when i came home i'will try. try to fork my project into yours

**Comment by wahyupermadie** [comment]
its okay, thank u so much sir

**Comment by alvinteeky** [comment]
Did the error fixed ? i have the same problem?

**Comment by Khuram007** [comment]
please comment to require('./bootstrap') on the second app.js file, I have 3 files 1) admin_app.js 2) admin_search.js and app.js ,this error show on multiple instances of the root.
![image](https://user-images.githubusercontent.com/36439994/103077047-dbfa1780-45f0-11eb-9d44-c35a6b2dcc84.png)
